.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@ZaricZoran @lancesix
```

Comment:
Should this change be done in the Gerrit git repo since that is the authorative source. It is pushed to this github.

Agree that @lmoriche should be included.

Reply:
Agreed. I will close this review and port to Gerrit.